Package.swift

```diff
@@ -14,9 +14,15 @@ let package = Package(
             targets: ["PrimerSDK"]
         )
     ],
+    dependencies: [
```

Comment:
These changes are the previously agreed changes to unblock their build. Not directly related